distribution/pull_v2.go

```diff
@@ -606,13 +618,28 @@ func (p *puller) pullSchema1(ctx context.Context, ref reference.Reference, unver
 	return imageID, manifestDigest, nil
 }
 
+func checkSupportedMediaType(mt string) error {
+	isOCIImageType := strings.HasPrefix(mt, "application/vnd.oci.image")
+	isDockerType := strings.HasPrefix(mt, "application/vnd.docker.")
```

Comment:
[HTTP media types are case-insensitive](https://httpwg.org/specs/rfc9110.html#field.content-type:~:text=The%20type%20and%20subtype%20tokens%20are%20case%2Dinsensitive.) so the checks also need to be.

Reply:
Thanks, good point! Fixed.